fhir-persistence-schema/docs/SchemaToolUsageGuide.md

```diff
@@ -0,0 +1,582 @@
+# IBM FHIR Server - Schema Deployment
+
+This document details how to deploy the IBM FHIR Server schema and apply release upgrades. This document also describes how the schema changes are managed.
+
+The schema tool generates the following object types that require management: 
+
+- DB2 PACKAGE
+- TABLESPACES
+- SCHEMA 
+- GLOBAL VARIABLE 
+- INDEX 
+- PERMISSION 
+- PROCEDURE 
+- SEQUENCE 
+- TABLE
+- TABLE CONSTRAINT 
+
+For details on the schema design, refer to the [Schema Design](https://github.com/IBM/FHIR/tree/master/fhir-persistence-schema/docs/SchemaDesign.md) document.
+
+----------------------------------------------------------------
+## Database Support
+
+
+| Database   | Version   | Support |
+|------------|-----------|-----------------------------------|
+| DB2        |      11.5 | Suitable for production. Supports multi-tenancy. |
```

Comment:
having "suitable for production" here and not in PostgreSQL still seems to imply this one is somehow more mature. 
```suggestion
| DB2        |      11.5 | Supports multiple tenants in the same db schema. |
```

Reply:
fixed